lib/ui/compositing.dart

```diff
@@ -272,10 +272,10 @@ class SceneHost extends NativeFieldWrapperClass2 {
   /// to the child.
   ///
   /// The scene host takes ownership of the provided export token handle.
-  SceneHost(int export_token_handle) {
+  SceneHost(dynamic export_token_handle) {
```

Comment:
Is there any documentation for dart:fidl.internal that I could read to educate myself about what's going on here? I don't feel I understand the problem space well enough to know how to solve this.

We definitely don't want to take `dynamic`, that loses all the type safety of the system, not to mention that it's a documentation dead-end so people can't follow a link to the next step.